test/test_matmul_cuda.py

```diff
@@ -204,10 +204,19 @@ def _expand_to_batch(t: torch.Tensor):
         self.assertEqual(out1_gpu, out2_gpu[0])
 
 
+def scaled_mm_supported_device():
+    return torch.cuda.is_available() and (
+        torch.cuda.get_device_capability >= (9, 0) or torch.cuda.get_device_capability() == (8, 9)
```

Comment:
typo:
```python
torch.cuda.get_device_capability >= (9, 0)
```
->
```python
torch.cuda.get_device_capability() >= (9, 0)
```

Reply:
Hmm, how it was not caught in the testing?  As comparing function to a tuple should raise type error
```
>>> range >= (2, 3)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: '>' not supported between instances of 'type' and 'tuple'
```

Reply:
For provenance: CI tests were failing